docs/dev/metrics.md

```diff
@@ -88,6 +88,26 @@ In addition, Prometheus provides some default metrics about the internal state
 of the running process and the metric collection. You can find more information
 about these metric names and their labels through the following links:
 
+## Machine API error rate for provider
```

Comment:
the placement of this section cuts the "Metrics about the Prometheus collectors" in half. this new section should be moved to line 73 at the least.